includes/admin.form.inc

```diff
@@ -35,6 +44,27 @@ function islandora_pathauto_admin_settings(array $form, array &$form_state) {
     '#tree' => TRUE,
     '#theme' => 'islandora_pathauto_admin_table',
   );
+  // Only add this option if the module is installed.
+  if (module_exists('islandora_solr')) {
+    $form['solr_options_wrapper'] = array(
+      '#type' => 'fieldset',
+      '#title' => 'Query Islandora objects settings',
+      '#description' => 'Options related to Solr search module being installed',
+    );
+    $form['solr_options_wrapper']['query_language'] = array(
+      '#type' => 'radios',
+      '#title' => t('Query language'),
+      '#options' => array_map($map_to_title, $backend_options),
+      '#description' => t('Query language to use for finding the content models in the system for pathauto hooks.'),
+      '#default_value' => variable_get('islandora_pathauto_query_language', 'islandora_basic_collection_sparql_query_backend'),
```

Comment:
I don't think we should be referencing 'islandora_basic_collection_sparql_query_backend' at all in this module; seems confusing.